src/xdocs/config_javadoc.xml

```diff
@@ -792,6 +832,103 @@ public int checkReturnTag(final int aTagIndex,
   &lt;property name="accessModifiers" value="private, package"/&gt;
 &lt;/module&gt;
         </source>
+        <p> Example:</p>
+        <source>
+/**
+ * @param number1 The first number
+ * @param number2 The second number
+ */
+public int addTwoNumbers(int number1, int number2){
+     return number1 + number2;                   // ok, no checks made for public methods
+}
+
+/**
+ * @param number1 The first number
+ * @param number2 The second number
+ */
+private int subtractTwoNumbers(int number1, int number2){
+    return number1 - number2;                    // violation, return tag missing
+}
+/**
+ * @param number1 The first number
+ */
+void printProduct(int number1, int number2){    // violation, param tag missing
+    System.out.println(number1 * number2);      // ok, no return tag required. Method is checked
+}                                               // since it is package-private by default
```

Comment:
Please add a new line between the methods. Also please add an example of violation for `package-private` method i.e. access modifier not specified explicitly.

Reply:
@Vyom-Yadav I have removed the access modifier `public` from the `class Test` that I have been using for other examples to cover the `package-private` method use-case

Reply:
> Please use one code example across all the configs. Don't worry if the code example becomes lengthy. A good example is at https://checkstyle.sourceforge.io/config_coding.html#VariableDeclarationUsageDistance_Examples

@Vyom-Yadav  I have made all the code examples the same for every property (except for one use case of `package-private`). Could you please look into it. Thanks